tensorrt_llm/_torch/pyexecutor/decoder.py

```diff
@@ -337,12 +328,13 @@ def _batch_decode(self, scheduled_requests: ScheduledRequests,
         new_tokens_host = new_tokens_device.to('cpu', non_blocking=True)
         decoder_event = torch.cuda.Event()
         decoder_event.record()
-        return DecoderState(
-            scheduled_requests=scheduled_requests,
-            logits=logits,
-            new_tensors_device={"new_tokens_device": new_tokens_device},
-            new_tensors_host={"new_tokens_host": new_tokens_host},
-            decoder_event=decoder_event)
+        return DecoderState(scheduled_requests=scheduled_requests,
+                            logits=logits,
+                            new_tensors_device=OrderedDict(
```

Comment:
`OrderedDict` does not seem necessary here. `dict` preserves insertion order starting from Python 3.6, and `OrderedDict` is only better at reordering elements which we don't need.

Reply:
Thanks, agreed. Fixed.